bin/reth/src/args/rpc_server_args.rs

```diff
@@ -140,7 +140,11 @@ pub struct RpcServerArgs {
     #[arg(long, value_name = "COUNT", default_value_t = constants::DEFAULT_MAX_TRACING_REQUESTS)]
     pub rpc_max_tracing_requests: u32,
 
-    /// Maximum number of logs that can be returned in a single response.
+    /// Maximum number of blocks that could be scanned per filter request. (0 = entire chain)
+    #[arg(long, value_name = "COUNT", default_value_t = constants::DEFAULT_MAX_BLOCKS_PER_FILTER)]
+    pub rpc_max_blocks_per_filter: u64,
+
+    /// Maximum number of logs that can be returned in a single response. (0 = no limit)
```

Comment:
I think we can do that, even though I'm a bit hesitant to abuse the 0 value.

Reply:
@mattsse I have used this since Geth supports flags with the similar method https://geth.ethereum.org/docs/fundamentals/command-line-options

Reply:
okay, let's interpret the 0 value like `None` then